packages/smart-router/evm/v3-router/providers/poolProviders/subgraphPoolProviders.ts

```diff
@@ -0,0 +1,220 @@
+import { ChainId, Currency } from '@pancakeswap/sdk'
+import { parseProtocolFees, Pool, FeeAmount } from '@pancakeswap/v3-sdk'
+import { gql } from 'graphql-request'
+import type { GraphQLClient } from 'graphql-request'
+import memoize from 'memoizee'
```

Comment:
how about use `lodash/memoize.js`?

Reply:
I'm thinking to get rid of lodash if possible

Reply:
What about we use `lodash-es` instead?

Reply:
you will need to move all lodash to avoid duplicate lodash + lodash-es import